### PR TITLE
Add "Report user" button at the bottom of the user profile

### PR DESCRIPTION
**Pull request by s-cheng**
This PR adds a button at the bottom of the user profile which lets other users mark them as `needsReview` if they have not already been reviewed.

This was requested by Lizka, as a way to help the moderators remove spam which does not always get surfaced through the existing channels.

Screen Shot 2022-05-31 at 8 53 02 PM

### CODE REVIEW

**Comment by s-cheng** [comment]
@darkruby501 This seemed fine to include in LW, but let me know if you want me to forum-gate it.

**Comment by jpaddison3** [comment]
> which does not always get surfaced through the existing channels

This is just tag spam right? Like, this won't allow you to report a user who slipped through and got approved when they shouldn't've been.

Maybe a better way is to allow the reporting of any user under N (=10?) karma?

**Comment by s-cheng** [comment]
@jpaddison3 I'm more concerned about spam in user profiles (especially if they get more visibility as we build related features). Lizka also raised the concern about spam via PMs, though I'm not sure if that has actually happened.

But really the main reason for the "not already been reviewed" condition is that it was the easiest. I think it's good that we have **a** way to report users, but I'm not sure if it's worth me spending any more time on this since it's pretty unclear how much of a problem this is.

File: packages/lesswrong/components/users/UsersProfile.tsx [hunk 1]
```diff
@@ -148,6 +150,10 @@ const styles = (theme: ThemeType): JssStyles => ({
   userMetaInfo: {
     display: "inline-flex"
   },
+  reportUserSection: {
+    textAlign: 'center',
```
Comment (jpaddison3) [inline packages/lesswrong/components/users/UsersProfile.tsx:154]:
```suggestion
```
I think our site aesthetic is to left align things like this.

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

**Pull request merged**